go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/opencontainers/selinux
 
-go 1.13
+go 1.18
 
-require golang.org/x/sys v0.0.0-20191115151921-52ab43148777
+require golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
```

Comment:
Might be good to do this in a separate commit (keeping the "fix" separate from assisting changes)